adapter/index.ts

```diff
@@ -1,37 +1,47 @@
 import { injectExtension } from '@polkadot/extension-inject';
-
-interface Injected {
-    // the interface for Accounts, as detailed below
-    readonly accounts: Accounts;
-    // the standard Signer interface for the API, as detailed below
-    readonly signer: Signer;
-    // not injected as of yet, subscribable provider for polkadot-js API injection,
-    // this can be passed to the API itself upon construction in the dapp
-    // readonly provider?: Provider
+import {Injected, InjectedAccounts} from "@polkadot/extension-inject/types";
+import { Signer as InjectedSigner } from '@polkadot/api/types';
+import {installPolkadotSnap, isPolkadotSnapInstalled} from "../example/src/services/metamask";
+import {pluginOrigin} from "../example/src/services/metamask";
```

Comment:
Don't import code from example, copy code here. Example will use this adapter later on